mitosheet/src/components/elements/BottomLeftPopup.tsx

```diff
@@ -35,31 +36,22 @@ const BottomLeftPopup = (props: {
         }
     }, [props.loading.length]);
 
-    // We only display the checklist if we are in variant a
-    const displayChecklist = isVariantA(props.analysisData) 
-        && getRemainingChecklistItems(props.userProfile).length > 0
-        && props.currOpenModal.type !== ModalEnum.SignUp;
-
     return (
         <>
             {displayLoadingIndicator && 
                 <div className='bottom-left-popup-container'>
                     <LoadingIndicator loading={props.loading}/>
                 </div>
             }
-            {!displayLoadingIndicator && displayChecklist &&
+            {!displayLoadingIndicator && props.ephemeralMessage &&
```

Comment:
So we're not doing a checklist on onboarding anymore? I don't know if I think that's a great idea. 

We AB tested tour vs checklist, but never checklist v nothing - and IMO a tour checklist is kinda nice for getting people to do something!

Reply:
We might do it at some point. But we are not currently doing it